src/library/setlogfeature.cpp

```diff
@@ -27,7 +27,7 @@ SetlogFeature::SetlogFeature(
                           nullptr,
                           pLibrary->trackCollections(),
                           "mixxx.db.model.setlog",
-                          /*show-all-tracks*/ true),
+                          true),
```

Comment:
Why did you remove this helpful comment? This comment reflected the fact that booleans are inappropriate as functions parameters in general. Such comments should not be removed without fixing the actual cause.

Especially in this case were setting it to false might cause the deletion of data from the database! ...which is bad and dangerous design decision imho, but that's another topic.

Reply:
Because I changed the parameter name to be more descriptive. Imo you should consult your IDE for such information about parameters, not thrown-in comments.

Reply:
For example, Intellij/CLion actually prefixes values in such calls with their name:
![image](https://user-images.githubusercontent.com/13354331/82540870-dd8dc980-9b4f-11ea-9d18-c33837447ef8.png)

Reply:
I disagree. The code must be comprehensible without an IDE. Please don't make such assumptions.

Reply:
I think that is a subjective general thing, so I'd like to get some more opinions. @Be-ing @daschuer @Holzhaus :)

@uklotzde would you then expect any boolean parameter to have such a comment? I feel like this ruins the point of sensible parameter naming.

Reply:
I'll be adding it back for now so we can merge, but I'd still like to talk about this.

Reply:
Boolean parameters don't read well at the call site. That's the point why we recommend to use enums instead in the future.

But there is no excuse for making the code worse without providing an alternative. This is not negotiable.

Reply:
@xerus2000 I already explained in detail why I think that the comment for this *particular* boolean parameter matters:

> Especially in this case were setting it to false might cause the deletion of data from the database!

Reply:
Moved: https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/About.20coding.20style/near/198313605